src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -155,3 +266,142 @@ def asset_download(**args):
     Downloads input manifest of previously submitted job.
     """
     click.echo("download complete")
+
+
+def read_local_manifest(manifest: str) -> BaseAssetManifest:
+    """
+    Read manifets specified by filepath to manifest folder, returns BaseAssetManifest Object
+    """
+    input_files = glob.glob(os.path.join(manifest, "*_input"))
+
+    if len(input_files) >= 2:
+        raise NonValidInputError(f"Does not support multiple manifest_input files in {manifest}. ")
```

Comment:
```suggestion
        raise NonValidInputError(f"Multiple input manifest files are not supported, but found {input_files}.")
```

nit: Slight improvement to the error message. Feel free to word as you want but listing the files we found is helpful